section/introduction.tex

```diff
@@ -1,22 +1,42 @@
 \section{Introduction}
- 
-The large-scale publication of linked data empowers more freedom in the creation and usage of web applications.
-Concretely linked data can diminish data silos \cite{Verstraete2022}
+\rv{Our target audience does not need the next paragraph.}
+
+\remove{
+The large-scale publication of Linked Data empowers more freedom in the creation and usage of web applications.
+\rv{I don't think the previous sentence is true.}
+Concretely Linked Data can diminish data silos \cite{Verstraete2022}
 and could foster potential new forms of application ownership \cite{Mechant2021}.
+\rv{Neither of those two claims is true. Linked Data is just a technology. It's an inanimate object; it can't make those changes.}
 The \href{https://solidproject.org/TR/protocol}{Solid} and
 \href{https://docs.joinmastodon.org/}{Mastodon} protocols
-are popular technologies utilizing the emancipatory potential of linked data.
-Linked data is mostly modeled and published using the graph data formalization \href{https://www.w3.org/TR/rdf12-concepts/}{RDF} and its serializations.
-RDF terms can be expressed using IRIs, Blank Nodes, and Litterals.
```

Comment:
I'm going to be very whiny now, but: avoid typos at all costs. (`Litterals`) They slow us down unnecessarily. Configure your editor to do spell checks, and don't commit anything with a red squiggly line.

It's understandable if there's typos in manuscripts that machines cannot catch. But don't leave the ones in that machines can catch. There's just no reason.